client/src/components/Modal/Modal.css

```diff
@@ -0,0 +1,125 @@
+.Modal {
+  width: 100%;
+  height: 309px;
+  margin: 0 auto;
+  padding: 1rem 2rem 0.5rem 2rem;
+  background: var(--yellow);
+  box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+  border-radius: 3px;
+  display: flex;
+  flex-direction: column;
+  font-family: var(--default-font);
+  font-weight: var(--title-weight);
+  color: black;
+  transition: all 0.8s;
+}
+
+.Modal__close {
+  text-align: right;
+}
+
+.Modal__closeButton {
+  font-size: 5rem;
+  line-height: 56px;
+  text-decoration: none;
+  color: black;
+}
+
+.Modal__body {
+  width: 100%;
+  height: 60%;
+}
+
+.Modal__messageBox {
```

Comment:
With the naming convention we're using, I think this should be `.Modal__message-box`

Reply:
(Sorry for not catching this earlier 😳 )